test/atm/global-workflow/jcb-prototype_3dvar-fv3inc.yaml.j2

```diff
@@ -4,7 +4,7 @@ algorithm: fv3jedi_fv3inc_variational
 
 # Testing things
 # --------------
-{% if DO_TEST_MODE and RUN == 'gdas'  %}
+{% if DO_TEST_MODE and RUN == 'gdas' and current_cycle | to_YMDH == '2021032318' %}
```

Comment:
I recommend not relying on Jinja's implicit boolean checks and instead to check them explicitly.  This is because the wxflow extension of Jinja will initialize uninitialized values to, e.g., `"{{ DO_TEST_MODE }}"`, which `if` will evaluate as `True`.  Instead, use `if DO_TEST_MODE == True and ....`.  I'm going to open a wxflow issue to see if this can be changed so undefined values will return `False`.

Reply:
After some wxflow testing, I see that this is not true (or no longer true).  That said, there is no harm in explicit boolean tests.

Reply:
Thanks @DavidHuber-NOAA !